runtime/moonbase/src/lib.rs

```diff
@@ -1829,6 +1829,6 @@ mod tests {
 	// Required migration is parachain_staking::migrations::IncreaseMaxDelegatorsPerCandidate
 	// Purpose of this test is to remind of required migration if constant is ever changed
 	fn updating_maximum_delegators_per_candidate_requires_configuring_required_migration() {
-		assert_eq!(MaxDelegatorsPerCandidate::get(), 500);
+		assert_eq!(MaxDelegatorsPerCandidate::get(), 300);
```

Comment:
Are these meant to change from 500 -> 300?

Reply:
they were changed in https://github.com/PureStake/moonbeam/commit/7a540e1a78ab4e333efaf1df7662aa1287d60aec by Alan (mistake push to master) but the tests were not updated